dm/dm/master/server_test.go

```diff
@@ -573,6 +573,41 @@ func (t *testMaster) TestWaitOperationOkRightResult(c *check.C) {
 	}
 }
 
+func (t *testMaster) TestStopTaskWithExceptRight(c *check.C) {
+	taskName := "test-stop-task"
+	notExceptResp := &pb.QueryStatusResponse{
+		SubTaskStatus: []*pb.SubTaskStatus{
+			{
+				Name: "task1",
+				Status: &pb.SubTaskStatus_Sync{Sync: &pb.SyncStatus{
+					UnresolvedGroups: []*pb.ShardingGroup{{Target: "`db`.`tbl`", Unsynced: []string{"table1"}}},
+				}},
+			},
+		},
+	}
+	exceptResp := &pb.QueryStatusResponse{SubTaskStatus: []*pb.SubTaskStatus{}}
```

Comment:
https://github.com/pingcap/ticdc/blob/9da1946522cb7d132ebc16138ef84746aff2e237/dm/dm/worker/status.go#L72-L76

why remove this response by git force push?

Reply:
misunderstand, unittest refactored